### ISSUE TITLE
Could "Image filename is accessible name for image" (9eb3f6) include encoding?

### ISSUE THREAD

**Issue by annethyme** [act-rules/act-rules.github.io#1489]
Image alt issue seen on a contact page and not caught as an issue by an accessibility conformance testing engine using ACT rules: 

<img  alt="Name Lastname BH2836_250p.png" src="../Name%20Lastname%20BH2836_250p.png">

I would intuitively have expected this to be caught by the rule "Image filename is accessible name for image" (9eb3f6), but because of the encoding, it isn't caught. 
Should be easy enough to automate in the check though, so it would be lovely if the rule could be adjusted to include this scenario as well.

**Comment by WilcoFiers** [comment]
I don't quite follow. Perhaps because the image you included is broken? Can you explain?

**Comment by carlosapaduarte** [comment]
This is the code for the image @annethyme posted

```
<img  alt="Name Lastname BH2836_250p.png" src="../Name%20Lastname%20BH2836_250p.png">
```

src and alt attributes are only different because src is URL encoded

**Comment by annethyme** [comment]
Rule has been deprecated
